test/templates/file-uploader/component.njk

```diff
@@ -0,0 +1,25 @@
+<!--
+file-uploader:
+    properties={},
+    modifier={ block: []},
```

Comment:
is this commented block needed?

Reply:
yes it is required. to just understand the signature and available properties.